libs/features/cdt/cdt.css

```diff
@@ -35,6 +35,12 @@
   display: flex;
 }
 
+html[dir="rtl"] .horizontal .timer-block {
+  margin-right: 10px;
+  margin-left: 0px;
+  flex-direction: row-reverse;
+}
+
 .horizontal .timer-block {
   margin-left: 10px;
```

Comment:
can't this rather be achieved with [margin-block-start](https://developer.mozilla.org/en-US/docs/Web/CSS/margin-block-start)?

Reply:
No, it added margin on top.
I have achieved the same using column-gap now, looks good?